include/linker/common-ram.ld

```diff
@@ -45,6 +45,19 @@
 	} GROUP_DATA_LINK_IN(RAMABLE_REGION, ROMABLE_REGION)
 #endif
 
+#if defined(CONFIG_PM_DEVICE_POWER_DOMAIN_DYNAMIC) && (CONFIG_PM_DEVICE_POWER_DOMAIN_DYNAMIC != 0)
+	SECTION_DATA_PROLOGUE(device_handles,,)
+	{
+		__device_handles_start = .;
```

Comment:
This common section for linking the device handles in ROM or RAM could be pulled out to a common file (device-handles.ld) that is included in both locations so we can't update one and miss the other

Reply:
let me look how it will looks like and come up with a new version.